packages/api/endpoints/granules.js

```diff
@@ -55,6 +61,48 @@ async function list(req, res) {
   return res.send(result);
 }
 
+/**
+ * Create new granule
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} promise of an express response object.
+ */
+const create = async (req, res) => {
+  const {
+    knex = await getKnexClient(),
+    granulePgModel = new GranulePgModel(),
+  } = req.testContext || {};
+
+  let collectionCumulusId;
+  const granule = req.body;
+
+  try {
+    const collectionNameVersion = deconstructCollectionId(granule.collectionId);
+    collectionCumulusId = await getCollectionCumulusId(collectionNameVersion, knex);
```

Comment:
Should we pre-emptively throw an error if `collectionCumulusId` is undefined somehow, since it's required for the PG write anyway?

Reply:
no since you don't need it anymore in this fxn.